pulseapi/entries/views.py

```diff
@@ -47,6 +47,13 @@ def post_validate(request):
 
     return True
 
+class EntriesPagination(PageNumberPagination):
+    """
+    Add support for pagination and custom page size
+    """
+    page_size = 48
+    page_size_query_param = 'page_size'
```

Comment:
Oh good point- I was going to change it originally but then decided to leave it. Removing.